region/multi.go

```diff
@@ -25,8 +25,14 @@ var multiPool = sync.Pool{
 }
 
 func freeMulti(m *multi) {
+	// Set pointers to nil to allow the objects to be garbage
```

Comment:
TIL. I originally thought "this can't be true, Go should be smart enough to realise that slice semantic doesn't allow us to access index >= len(slice) and so it should GC them", but no :(